docs/faq.md

```diff
@@ -0,0 +1,261 @@
+| [API keys](#api-keys) |       <ul><li>[How do I get my Self-Service API key?](#how-do-i-get-my-self-service-api-key)</li><li>[Why is my Self-Service API key not working?](#why-is-my-self-service-api-key-not-working)</li><li>[How long is my Self-Service access token valid for?](#how-long-is-my-self-service-access-token-valid-for)</li><li>[Can I use my API key in a public repository?](#can-i-use-my-api-key-in-a-public-repository)</li><li>[Why has my API key been revoked?](#why-has-my-api-key-been-revoked)</li></ul>              |
+| [Billing](#billing) |         <ul><li>[How is billing calculated for Self-Service APIs?](#how-is-billing-calculated-for-self-service-apis)</li><li>[How do I request a refund of my Self-Service usage bill?](#how-do-i-request-a-refund-of-my-self-service-usage-bill)</li></ul>            |
+| [Test collection](#test-collection) |           <ul><li>[Is there a limit to the calls I can make to Self-Service APIs in the test environment?](#is-there-a-limit-to-the-calls-i-can-make-to-self-service-apis-in-the-test-environment)</li><li>[What should I do if I'm about to reach my Self-Service free request quota limit?](#what-should-i-do-if-im-about-to-reach-my-self-service-free-request-quota-limit)</li><li>[Is there a limit to the calls I can make to Self-Service APIs in the production environment?](#is-there-a-limit-to-the-calls-i-can-make-to-self-service-apis-in-the-production-environment)</li><li>[Why do I get a 429 error in JSON if I have some free calls left?](#why-do-i-get-a-429-error-in-json-if-i-have-some-free-calls-left)</li><li>[Why do I get an error code 429 when I call a Self-Service API?](#why-do-i-get-an-error-code-429-when-i-call-a-self-service-api)</li><li>[Is the data returned in the Self-Service test environment accurate?](#is-the-data-returned-in-the-self-service-test-environment-accurate)</li></ul>           |
+| [Flight Inspiration Search](#flight-inspiration-search) |       <ul><li>[Why didn't I get any results for Flight Inspiration Search?](#why-didnt-i-get-any-results-for-flight-inspiration-search)</li></ul>           |
+| [Airport Nearest Relevant API](#airport-nearest-relevant-api) |       <ul><li>[Why isn't the Airport Nearest Relevant API returning a specific airport near me?](#why-isnt-the-airport-nearest-relevant-api-returning-a-specific-airport-near-me)</li></ul>           |
+| [Flight Create Orders API](#flight-create-orders-api) |        <ul><li>[How are tickets issued for flights booked with Flight Create Orders in Self-Service?](#how-are-tickets-issued-for-flights-booked-with-flight-create-orders-in-self-service)</li><li>[How can I retrieve booking made with Flight Create Orders in Self-Service?](#how-can-i-retrieve-booking-made-with-flight-create-orders-in-self-service)</li><li>[Does Amadeus pay a commission for flights booked with Flight Create Orders in Self-Service?](#does-amadeus-pay-a-commission-for-flights-booked-with-flight-create-orders-in-self-service)</li></ul>                 |
```

Comment:
Even though this is specific API related, I would keep it here since flight booking is one of the top features developers are looking for.